src/main/command/birthdaycommands/DeleteBirthdayCommand.ts

```diff
@@ -0,0 +1,49 @@
+import { MessageEmbed } from 'discord.js';
+import { Command } from '../Command';
+import { CommandResult } from '../classes/CommandResult';
+import { CommandArgs } from '../classes/CommandArgs';
+import { deleteBirthday } from '../../modules/birthday/BirthdayDbUtil';
+
+const SUCCESSFUL_COMMANDRESULT = new CommandResult(true);
+const UNSUCCESSFUL_COMMANDRESULT = new CommandResult(false);
+
+export class DeleteBirthdayCommand extends Command {
+    public static readonly NAME = 'DeleteBirthday';
+
+    public static readonly DESCRIPTION = "Deletes the user's birthday.";
+
+    /**
+     * Displays all the users with a birthday on the date given as an argument.
+     *
+     * @param { CommandArgs } commandArgs
+     * @returns CommandResult
+     */
+    public async execute(commandArgs: CommandArgs): Promise<CommandResult> {
+        const { userId, messageReply, server } = commandArgs;
+
+        if (!userId) {
```

Comment:
`userId` will be truthy. It's the author of the message sent. Just assert it using the `!` operator, then we don't need the invalid embed part already.

Reply:
I made it optional in `CommandArgs` so I could use dependency injection for testing 😬